db/migration/20210430212322_add_bpm_metadata.go

```diff
@@ -0,0 +1,31 @@
+package migrations
+
+import (
+	"database/sql"
+
+	"github.com/pressly/goose"
+)
+
+func init() {
+	goose.AddMigration(upAddBpmMetadata, downAddBpmMetadata)
+}
+
+func upAddBpmMetadata(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+alter table media_file
+add bpm integer;
```

Comment:
As this field will appear in the Datagrids, it will be sortable, so we should add a index for it.